web/src/components/storage/ProposalPage.tsx

```diff
@@ -78,9 +82,13 @@ function InvalidConfigEmptyState(): React.ReactNode {
         </List>
       </EmptyStateBody>
       <EmptyStateFooter>
-        <Content component="p">{_("Do you want to reset to the default settings?")}</Content>
+        <Content component="p">
+          {_(
+            "You may want to discard those settings and start from scratch with a simple configuration.",
```

Comment:
those or these?

Reply:
I would say "those" because the settings are not really displayed. And the reference to them is at a previous paragraph. Both sound like "distance" to me.